webextension/background.js

```diff
@@ -298,9 +321,6 @@ const tabPageCounter = {
   counter: {},
 
   init() {
-    browser.tabs.onCreated.addListener(this.initTabCounter.bind(this));
-    browser.tabs.onRemoved.addListener(this.sendTabCountAndDelete.bind(this));
-    browser.webRequest.onCompleted.addListener(this.incrementTabCount.bind(this), {urls: ["<all_urls>"], types: ["main_frame"]});
```

Comment:
Yup will remove.

Still managed to get this:
```
Extension error: TypeError: this.counter[tabId] is undefined moz-extension://404995e8-9564-4f77-a5f6-41be189beb28/background.js 337
[[Exception stack
sendTabCountAndDelete@moz-extension://404995e8-9564-4f77-a5f6-41be189beb28/background.js:337:7
init/<@moz-extension://404995e8-9564-4f77-a5f6-41be189beb28/background.js:266:7
Current stack
runSafeSyncWithoutClone@resource://gre/modules/ExtensionUtils.jsm:73:129
runSafeSync@resource://gre/modules/ExtensionUtils.jsm:104:37
runSafe@resource://gre/modules/ExtensionCommon.jsm:123:32
receiveMessage@resource://gre/modules/ExtensionChild.jsm:586:49
_callHandlers/<@resource://gre/modules/MessageChannel.jsm:574:17
_callHandlers@resource://gre/modules/MessageChannel.jsm:573:14
_handleMessage/deferred.promise<@resource://gre/modules/MessageChannel.jsm:637:7
_handleMessage@resource://gre/modules/MessageChannel.jsm:634:24
receiveMessage@resource://gre/modules/MessageChannel.jsm:165:5
]]
```

Not sure on STR, I assume I can call a remove on a tab that wasn't counted somehow.